packages/admin/admin/src/form/FieldContainer.tsx

```diff
@@ -82,10 +84,13 @@ export const Root = styled(FormControl, {
         }
 
         ${ownerState.variant === "horizontal" &&
+        !ownerState.forceVertical &&
         css`
+            container-type: inline-size;
             display: flex;
             flex-direction: row;
             align-items: center;
+            max-width: 944px;
```

Comment:
What is this arbrtrary number?

Reply:
This number is mentioned in the ticket like that.

Reply:
@jamesricky should this be part of the theme?

Reply:
This number was defined by UX, I think it's the width form's have, when they are inside a Dialog, with the default size. They decided to make this the maximum width in general. 

I don't think we need to do anything in the theme for that, for the rare case, that this should be `100%` or something else, we can use the `styleOverrides` in the project's theme to change this.

Reply:
If this is the max width in general, then shouldn't it also be applied for `variant="vertical"`? And should we add a comment or define a constant?

Reply:
> If this is the max width in general, then shouldn't it also be applied for variant="vertical"?

This would cause an issue where the fields would unintentionally be rendered next to each other, if the form/window is twice the width of the fields, as they have `display: inline-flex` unlike the `display: flex` of the `horizontal` variant. 

But according to the UX-Team, the `vertical` variant might be phased out in general, as with this PR the fields are shown in a more space-efficient way, depending on the width they have available. 
So it might not be worth the effort to optimize the `vertical` variant.

Reply:
> And should we add a comment or define a constant?

I don't think this is necessary, as it's defined in the UX-Teams design guidelines 🤷🏼